run/endpoints-v2-backend/README.md

```diff
@@ -0,0 +1,49 @@
+# Cloud Run Endpoints Sample
+
+This sample shows how to create a Cloud Endpoints service.
+
+Use it with the [Getting Started with Endpoints for Cloud Run with ESPv2 tutorial][tutorial_link].
+
+For more details on how to work with this sample read the [Google Cloud Run Java Samples README][tutorial_link].
+
+[![Run in Google Cloud][run_img]][run_link]
+
+## Dependencies
+
+* **Spring Boot**: Web server framework.
+* **SpringDoc OpenApi**: API documentation generation tool.
+* **Jib**: Container build tool.
+
+## Build the Container Image
+
+1. Update `PROJECT_ID` in `pom.xml` with your GCP Project Id:
+
+    ```
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <endpoints.project.id>PROJECT ID</endpoints.project.id>
+    </properties>
+    ```
+
+1. Update `<YOUR_HOST_HERE>` and `<YOUR_APP_URI_HERE>` in EndpointsApplication.java.

```

Comment:
Need more information for these variables. Probably should add that the users needs to follow this section first [Reserving a Cloud Run hostname](https://cloud.google.com/endpoints/docs/openapi/get-started-cloud-run#reserve_hostname)